content/browser/ml/webnn/dml/execution_resources.cc

```diff
@@ -51,11 +47,19 @@ ComPtr<ID3D12Resource> ExecutionResources::Allocate(UINT64 resource_size) {
     return nullptr;
   }
 
-  ComPtr<ID3D12Resource> resource;
+  RESOURCE_PTR resource;
+#ifdef ENABLE_GPU_MEMORY_MANAGEMENT
   // D3D12 creates an implicit heap that contains the resource allocation when
```

Comment:
We can remove these comments. From the perspective of DML, it does not care where or how memory was allocated.